src/sentry/static/sentry/app/components/loadingIndicator.tsx

```diff
@@ -82,4 +80,6 @@ LoadingIndicator.propTypes = {
   hideSpinner: PropTypes.bool,
 };
 
-export default withProfiler(LoadingIndicator);
+export default withProfiler(LoadingIndicator, {
+  hasUpdateSpan: false,
```

Comment:
mmm I can't form words atm, but `has` doesn't read to me that this is an option for withProfiler... maybe something like `includeUpdateSpan`?

Reply:
Or just `updateSpan`

Reply:
`updateSpan` makes it sound like the profiler will perform an update of a span -- maybe `includeUpdates`?

Reply:
Something like
```
// If time component is on page should be displayed as a span. True by default.
includeRender?: boolean;
// If component updates should be displayed as spans. True by default.
includeUpdates?: boolean;
```

Reply:
For now I am going to merge this in, but I will make the API changes and come back and update.